python/lsst_versions/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""Version handling following LSST coding conventions."""
```

Comment:
A docstring so that pydoc works:

```
$ pydoc lsst_versions

Help on package lsst_versions:

NAME
    lsst_versions - Version handling following LSST coding conventions.
```

I'll add the introductory text you suggested to the file so that that appears as well. I will add the license but note that if you just have the license then `pydoc` will actually read the license as the description (see `pydoc lsst` for example).

It's a bit annoying that it does not include the exported functions in the help string though.

Reply:
I've worked out how to make `pydoc lsst_versions` report the functions.